app/views/handlers/feedback.py

```diff
@@ -266,27 +292,105 @@ def __init__(
         self.feedback_type = feedback_type
 
     def __call__(self) -> dict[str, Any]:
+        # type ignores added as metadata will exist at this point
```

Comment:
Same here about type ignores.

Reply:
Updated 👍

Reply:
Metadata is no longer optional so the `if` statements are now redundant.